manifests/4.3/local-storage-operator.v4.3.0.clusterserviceversion.yaml

```diff
@@ -29,7 +29,7 @@ metadata:
       ]
     categories: Storage
     capabilities: Full Lifecycle
-    containerImage: quay.io/openshift/origin-local-storage-operator:4.3.0
+    containerImage: quay.io/openshift/origin-local-storage-operator:latest
```

Comment:
I thought this is only for display purpose in the UI. The actual image version is mentioned below which does get replaced by the ART pipeline.

Reply:
This is still a good idea to do but this may not fix the bug that was reported.

Reply:
We discussed this on slack. I'm reaching out to the reporter of the issue to confirm that the images are still correct, and that it's simply the `containerImage` reporting the tag instead of the digest.

I'm leaving this as is for now, as the reported issue involves fetching the `containerImage` and it reporting an incorrect tag.